releasenotes/releasenotes_v5.rst

```diff
@@ -13,6 +13,24 @@ Current Firely Server release notes (v5.x)
 Release 5.7.0, May 29th, 2024
 -----------------------------
 
+Features
+^^^^^^^^
+
+Improvements
+^^^^^^^^^^^^
+#. Configuration values logged on startup are now redacted based on the principle of inclusion versus exclusion. This change may cause custom configuration values to be hidden in the log output.
```

Comment:
Why not advise here that customers relying on this for a plugin need to log the config values on their own?